pkgs/development/libraries/nghttp3/default.nix

```diff
@@ -26,6 +27,10 @@ stdenv.mkDerivation rec {
   doCheck = true;
   enableParallelBuilding = true;
 
+  passthru.tests {
```

Comment:
typo.

Reply:
I should not just cherry-pick changes from old branches of mine